packages/flutter/lib/src/foundation/_ui_stub.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
```

Comment:
This probably needs a doc comment since I believe IDEs will send you here when you click-through?